python/persistenceClustering1.py

```diff
@@ -0,0 +1,124 @@
+from paraview.simple import *
+
+# create a new 'CSV Reader'
+clusteringcsv = CSVReader(FileName=["clustering1.csv"])
+
+# create a new 'Table To Points'
+tableToPoints1 = TableToPoints(Input=clusteringcsv)
+tableToPoints1.XColumn = "X"
+tableToPoints1.YColumn = "Y"
+tableToPoints1.ZColumn = "X"
+tableToPoints1.a2DPoints = 1
+tableToPoints1.KeepAllDataArrays = 1
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints2 = TTKIcospheresFromPoints(Input=tableToPoints1)
+tTKIcospheresFromPoints2.Radius = 0.05
+
+# create a new 'Gaussian Resampling'
+gaussianResampling1 = GaussianResampling(Input=tableToPoints1)
+gaussianResampling1.ResampleField = ["POINTS", "ignore arrays"]
+gaussianResampling1.ResamplingGrid = [256, 256, 3]
+gaussianResampling1.SplatAccumulationMode = "Sum"
+
+# create a new 'Slice'
+slice1 = Slice(Input=gaussianResampling1)
+slice1.SliceType = "Plane"
+slice1.HyperTreeGridSlicer = "Plane"
+slice1.SliceOffsetValues = [0.0]
+
+# init the 'Plane' selected for 'SliceType'
+slice1.SliceType.Origin = [-0.0187499550000001, 0.0313220950000002, 0.0]
+slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=slice1)
+tTKPersistenceDiagram1.ScalarField = ["POINTS", "SplatterValues"]
+tTKPersistenceDiagram1.InputOffsetField = [None, ""]
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.